src/headless/plugins/chat/model.js

```diff
@@ -731,6 +731,8 @@ const ChatBox = ModelWithContact.extend({
                 'id': origin_id,
                 'xmlns': Strophe.NS.FASTEN
             }).c('retract', {xmlns: Strophe.NS.RETRACT})
+            .c('fallback', {xmlns: Strophe.NS.FALLBACK}).up()
+            .c('body').t("This person attempted to retract a previous message, but it's unsupported by your client.");
```

Comment:
@jcbrand, I changed this to ".t('This person attempted to retract a previous message, but it is unsupported by your client.');" and made the relevant changes to the tests as well. The error persists. i.e. :-

`Error: Expected '<message id="f24d5790-9d3d-491b-81cb-e629c85899c9" to="mercutio@montague.lit" type="chat" xmlns="jabber:client"><store xmlns="urn:xmpp:hints"/><apply-to id="9db06479-12a8-4078-a0ec-96b289002f57" xmlns="urn:xmpp:fasten:0"><retract xmlns="urn:xmpp:message-retract:0"><fallback xmlns="urn:xmpp:fallback:0"/><body>This person attempted to retract a previous message, but it&apos;s unsupported by your client.</body></retract></apply-to></message>' to be '<message id="f24d5790-9d3d-491b-81cb-e629c85899c9" to="mercutio@montague.lit" type="chat" xmlns="jabber:client"><store xmlns="urn:xmpp:hints"/><apply-to id="9db06479-12a8-4078-a0ec-96b289002f57" xmlns="urn:xmpp:fasten:0"><retract xmlns="urn:xmpp:message-retract:0"/></apply-to><fallback xmlns="urn:xmpp:fallback:0"/><body>This person attempted to retract a previous message, but it is unsupported by your client.</body></message>'.`